composer.json

```diff
@@ -3,6 +3,9 @@
     "description": "The Zest Framework.",
     "keywords": ["framework", "zest"],
     "license": "MIT",
+    "require": {
+        "php": "^7.2"
```

Comment:
According to the `.travis.yml` setting, I think this framework require `php-7.2` version at least.

And we need to define the `php-7.2` version in `composer.json`.

Reply:
Yes you are right